build/FrameworkPackage/UpdateAppxFilesInNupkg.ps1

```diff
@@ -0,0 +1,137 @@
+<#
```

Comment:
Not sure if we need to copy this file too.

Reply:
not sure, this is going to be done in the next deliverable (test infra/testing) which has not yet been scheduled/assigend.  I fixed up some MUX/xaml names and left an undone comment for a path that is specific to xaml.